lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1513,6 +1513,11 @@ void SwiftASTContext::AddExtraClangArgs(const std::vector<std::string> &source,
   llvm::SmallString<128> cur_working_dir;
   llvm::SmallString<128> clang_argument;
   for (const std::string &arg : source) {
+    if (clang_argument == "-triple") {
```

Comment:
Can you add a comment explaining why we are filtering it out here?

Reply:
does the comment suffice?